app/configurations/config.hsl.js

```diff
@@ -751,6 +751,7 @@ export default {
   startSearchFromUserLocation: true,
 
   navigationLogo: 'hsl/navigator-logo.svg',
+  thumbsUpGraphic: 'hsl/thumbs-up.svg',
```

Comment:
We are currently using our icons in` svg-sprite.hsl.svg `and `svg-sprite.default.svg` We should move these icons there. Using a same key would render different images based on what instance is running.(And none if there is no corresponding icon, but may throw an error, not sure) 

I highly suspect we would want images for other instances as well, even though they are yet to be designed.

Reply:
I'd say this is out of scope of this PR